src/main/java/org/cactoos/func/Async.java

```diff
@@ -45,6 +45,10 @@
  * @param <X> Type of input
  * @param <Y> Type of output
  * @since 0.10
+ *
+ * @todo #1406:30m This class should not implement Proc, as it's always
+ *  used as Func. Clarify the rationale for it being Proc, if it's
+ *  unnecessary de-implement Proc. And implement Async for Proc if needed.
```

Comment:
@andreoss no need for this todo. This class can be used as a `Proc` so it can be useful for some users.

Reply:
@victornoel Done